idea/frontend/src/features/voucher/requests.ts

```diff
@@ -0,0 +1,13 @@
+import { INDEXER_RPC_SERVICE } from '@/shared/services/rpcService';
+
+import { PaginationResponse } from '@/api';
+
+import { METHOD } from './consts';
+import { Voucher, VouchersParams } from './types';
+
+const getVoucher = (id: string) => INDEXER_RPC_SERVICE.callRPC<Voucher>(METHOD.DATA, { id });
```

Comment:
As stated above, let's move it to `/api` folder, and also let's remove previous implementation from `/utils`.